src/main/scala/ScalaModulePlugin.scala

```diff
@@ -147,6 +147,17 @@ object ScalaModulePlugin extends AutoPlugin {
 
   lazy val scalaModuleSettingsJVM: Seq[Setting[_]] = scalaModuleOsgiSettings
 
+  lazy val scalaModuleSettingsNative: Seq[Settings[_]] = Seq(
+    scalaVersion := "2.11.11",
+    skip in compile := System.getProperty("java.version").startsWith("1.6"),
+    test := {},
+    libraryDependencies := {
+      if (!scalaVersion.value.startsWith("2.11"))
+        libraryDependencies.value.filterNot(_.organization == "org.scala-native")
+      else libraryDependencies.value
+    }
```

Comment:
sbt-doge was created to experiment with how to solve the problem, and wasn't merged into sbt 0.13 to avoid a breaking anyone relying on the old semantics. It has since been merged into sbt 1.